lighthouse-core/test/audits/offline-start-url-test.js

```diff
@@ -68,6 +68,8 @@ describe('Offline start_url audit', () => {
     assert.strictEqual(result.score, 1);
     assert.strictEqual(result.explanation, undefined);
     assert.strictEqual(result.warnings.length, 1);
-    assert.ok(result.warnings[0].includes('start_url must be same-origin as document'));
+    expect(result.warnings[0]).toBeDisplayString('Lighthouse couldn\'t read the start_url ' +
+    'from the manifest. As a result, the start_url was assumed to be the document\'s URL. ' +
+    'Error message: \'ERROR: start_url must be same-origin as document\'.');
```

Comment:
definitely a risk of making tests too noisy on every future code change like this (this test was just making sure that the proper manifestWarning made it through to the audit result, not that the string was exactly as written in the string literal). In this case, if the UIString changes how many other places will also need to be updated?

OTOH, we rarely update UIStrings, so this change seems fine if you really want to expand it.

Reply:
I guess when 2 people call it out a pattern emerges.  I'll relax them.